### PR TITLE
add new colors

### PR DESCRIPTION
**Pull request by uriash83**

### CODE REVIEW

**Comment by uriash83** [comment]
I've added some new colors

File: feature1/feature1.txt [hunk 1]
```diff
@@ -5,3 +5,6 @@ black
 yellow
 gray
 green
+brown
```
Comment (marcinmentel) [inline feature1/feature1.txt]: remove the brown color

**Review by marcinmentel — CHANGES_REQUESTED** [review]
Please change colors

**Review by uriash83 — COMMENTED** [review]
@marcinmentel 
brown color removed

**Review by uriash83 — COMMENTED** [review]
correted

**Review by uriash83 — COMMENTED** [review]
ok

**Review by uriash83 — COMMENTED** [review]
ok

**Review by marcinmentel — APPROVED** [review]

**Pull request merged**